### ISSUE TITLE
A11y violations across pages

### ISSUE THREAD

**Issue by tiagodread** [ansible/awx#10563]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
This is another round of a11y bugs found by cypress-axe engine, the .json report is attached

##### STEPS TO REPRODUCE

Install Axe Chrome extension, and run analyze for the page in question

##### EXPECTED RESULTS

No a11y violations

Report:
[a11yReport.zip](https://github.com/ansible/awx/files/6743654/a11yReport.zip)

**Comment by tiagodread** [comment]
Ok, ran the checks locally again and now based on our test coverage they look good.
